src/twisted/internet/test/test_process.py

```diff
@@ -862,6 +862,8 @@ class PTYProcessTestsBuilder(ProcessTestsBuilderBase):
     if platform.isWindows():
         skip = "PTYs are not supported on Windows."
     elif platform.isMacOSX():
+        skip = "PTYs are flaky from a Darwin bug. See #8840."
```

Comment:
To be clear, we can make re-enabling the PTY tests that _do_ work a separate ticket.  We don't need it here.  Please just file it so that we don't forget.